src/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/Components/UploadPhotosForm.tsx

```diff
@@ -65,7 +65,7 @@ export const UploadPhotosForm: React.FC<UploadPhotosFormProps> = ({
 
   useEffect(() => {
     const imagesToUpload = values.photos.filter(
-      c => !(c.geminiToken || c.url) && !c.loading
+      c => !(c.geminiToken || c.url) && !c.loading && !c.errorMessage
```

Comment:
Without filtering out artworks with an error message, this effect will try to re-add images that could not be loaded, which can lead to an infinite loop.

Alternatively, we could add a retry counter and only retry to upload an image 3 times. But I've decided against this solution because for me it seems to be too complex. The user can also just reupload the image manually.